src/Pelagos/Bundle/AppBundle/Resources/views/DatasetSubmission/qtip.html.twig

```diff
@@ -1,63 +1,49 @@
 
 <div id="regid_tip" style="display:none;">
-    {% image '@PelagosAppBundle/Resources/public/images/info.png' %}
-    <img class="info" src="{{ asset_url }}"  style="float:right;">
-    {% endimage %}
+    <img class="info" src="{{ asset('build/images/info32.png') }}">
```

Comment:
do you want to add the style=float right back?

Reply:
I almost want to make this qtip into it's own template. But I think most have the style added to the class.

Reply:
I added it, but doesn't seem to make a difference.